scripts/demo.tsx

```diff
@@ -205,6 +256,31 @@ app.get("/create", (c) => {
   );
 });
 
+app.get("/view/:id", (c) => {
+  const id = c.req.param("id");
+  if (!sv.ids.includes(id)) {
+    c.status(404);
+    return c.html(
+      <Template>
+        <h1>404 - Not Found</h1>
+        <p>The requested ID does not exist.</p>
+      </Template>,
+    );
+  }
+  return c.html(
+    <Template>
+      <iframe
+        src={`${sv.url}/${id}`}
+        style={{
+          width: "100%",
+          height: "100vh",
+          border: "none",
+        }}
+      />
+    </Template>,
```

Comment:
Instead of wrapping this in an iframe, we could also just return `fetch(sv.url + id)`

Reply:
I think it's chill, let's merge and we can always follow up later.